Formula/global.rb

```diff
@@ -43,8 +40,11 @@ class Global < Formula
     sha256 "a18f47b506a429f6f4b9df81bb02beab9ca21d0a5fee38ed15aef65f0545519f"
   end
 
+  # use homebrew sqlite instead of the older copy included in libdb/
+  patch :DATA
+
   def install
-    system "sh", "reconf.sh" if build.head?
+    system "sh", "reconf.sh"
```

Comment:
```suggestion
    system "autoreconf", "-svfi"
```
Since you're only patching autotools stuff, just calling `autoreconf` with the same arguments as in `reconf.sh`:
```
echo "- Generating configure items..."
(set -x; autoreconf --symlink --verbose --force --install) &&
```
should be sufficient, and avoids unnecessarily depending on `flex` and `bison` in Linux.